scripts/asm-installer/release_asm_installer

```diff
@@ -158,20 +189,31 @@ write_and_upload() {
   local FILE_PATH; FILE_PATH="${BUCKET_PATH}/${FILE_NAME}"
   local FILE_URI; FILE_URI="${BUCKET_URL}/${FILE_PATH}"
 
-  if ! changes_necessary "${FILE_URI}" "${FILE_NAME}"; then return; fi
+  if changes_necessary "${FILE_URI}" "${FILE_NAME}"; then
+    sha256sum "${SCRIPT_NAME}" >| "${SCRIPT_NAME}.sha256"
+    cp "${SCRIPT_NAME}" "${FILE_NAME}"
+    sha256sum "${FILE_NAME}" >> "${SCRIPT_NAME}.sha256"
+
+    upload "${SCRIPT_NAME}" "${FILE_NAME}" "${FILE_PATH}" "${FILE_URI}"
+
+    if [[ "${BRANCH_NAME}" == "${CURRENT_RELEASE}" ]]; then
+      upload "${SCRIPT_NAME}" "${FILE_NAME}" "${BUCKET_PATH}/${SCRIPT_NAME}" "${BUCKET_URL}/${BUCKET_PATH}/${SCRIPT_NAME}"
+    fi
 
-  sha256sum "${SCRIPT_NAME}" >| "${SCRIPT_NAME}.sha256"
-  cp "${SCRIPT_NAME}" "${FILE_NAME}"
-  sha256sum "${FILE_NAME}" >> "${SCRIPT_NAME}.sha256"
+    git restore "${SCRIPT_NAME}"
 
-  upload "${SCRIPT_NAME}" "${FILE_NAME}" "${FILE_PATH}" "${FILE_URI}"
+    append_version "$(./"${SCRIPT_NAME}" --version)" "${FILE_NAME}"
```

Comment:
At first, I don't think we want to do this every time, just if it's on the release branch. i.e. whenever we check for a git tag.

Reply:
So you mean we don't append the versions if it's unstable, staging, etc.?

Reply:
Right

Reply:
Done